core/src/main/java/tc/oc/pgm/community/commands/ModerationCommands.java

```diff
@@ -546,30 +532,27 @@ public void alts(
       int pages = (altAccounts.size() + perPage - 1) / perPage;
 
       Component pageNum =
-          new PersonalizedTranslatable(
-                  "command.simplePageHeader",
-                  new PersonalizedText(Integer.toString(page)).color(ChatColor.DARK_AQUA),
-                  new PersonalizedText(Integer.toString(pages)).color(ChatColor.DARK_AQUA))
-              .getPersonalizedText()
-              .color(ChatColor.GRAY);
+          TranslatableComponent.of("command.simplePageHeader", TextColor.GRAY)
+              .args(
+                  TextComponent.of(Integer.toString(page), TextColor.DARK_AQUA),
+                  TextComponent.of(Integer.toString(pages), TextColor.DARK_AQUA));
 
       Component headerText =
-          new PersonalizedTranslatable("moderation.alts.header")
-              .getPersonalizedText()
-              .color(ChatColor.DARK_AQUA);
+          TranslatableComponent.of("moderation.alts.header", TextColor.DARK_AQUA);
 
       Component header =
-          new PersonalizedText(
-              headerText,
-              new PersonalizedText(" (").color(ChatColor.GRAY),
-              new PersonalizedText(Integer.toString(altAccounts.size())).color(ChatColor.DARK_AQUA),
-              new PersonalizedText(") » Page ").color(ChatColor.GRAY),
-              pageNum);
+          TextComponent.builder()
+              .append(headerText)
+              .append(" (", TextColor.GRAY)
+              .append(Integer.toString(altAccounts.size()), TextColor.DARK_AQUA)
+              .append(") » Page ", TextColor.GRAY)
```

Comment:
Can we use the page translations we have? It's either under `command` or `misc`.